### ISSUE TITLE
Highlighting of property/methods in CoffeeScript

### ISSUE THREAD

**Issue by meleyal** [altercation/vim-colors-solarized#38]
Not sure if this is a `solarized` or `vim-coffee-script` (@kchmck) issue, but in vim all properties / method calls appear in a harsh red, while in TextMate + SublimeText they are a friendly blue:
#### Vim:

![Vim](http://meleyal.flxd.it/gup7_512.jpg)
#### Sublime Text:

![Sublime Text](http://meleyal.flxd.it/gup8_512.jpg)
#### TextMate:

![TextMate](http://meleyal.flxd.it/gup9_512.jpg)

**Comment by jamesmacwilliam** [comment]
try colorscheme desert
much more friendly

**Comment by jonahkagan** [comment]
I was having a similar issue, but with `this`. Looking through [vim-coffee-script/syntax/coffee.vim](https://github.com/kchmck/vim-coffee-script/blob/master/syntax/coffee.vim), I found these lines:

```
" A special variable
syn match coffeeSpecialVar /\<\%(this\|prototype\|arguments\)\>/ display
hi def link coffeeSpecialVar Special

" An @-variable
syn match coffeeSpecialIdent /@\%(\I\i*\)\?/ display
hi def link coffeeSpecialIdent Identifier
```

Identifiers are colored blue by the solarized syntax file, so I just changed `Special` to `Identifier` and that solved my problem.

I'm not sure why you're having the inverse issue, but maybe a similar approach could help.
